versioned_docs/version-24.04/administration/centreon-ha/operating-guide.md

```diff
@@ -0,0 +1,800 @@
+7       system user             NULL    Connect 0       Connecting to master    NULL    0.000
+8       system user             NULL    Slave_SQL       0       Slave has read all relay log; waiting for more updates    NULL    0.000
+```
+
+Database replication should now be OK. Check this.
+
+```shell
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+The result should be:
+
+```text
+Connection MASTER Status '@CENTRAL_NODE2_NAME@' [OK]
+Connection SLAVE Status '@CENTRAL_NODE1_NAME@' [OK]
+Slave Thread Status [OK]
+Position Status [OK]
+```
+
+Now, you can perform a failover to return to the initial situation.
+
+```shell
+pcs resource clear centreon
+```
+
+Do a cleanup to clear errors and restart the ms_mysql resource on @CENTRAL_NODE1_NAME@.
+
+```shell
+pcs resource cleanup
+```
+
+The situation has stabilized, and you can perform a failover by moving the **centreon** resource.
+
+```shell
+pcs resource move centreon
+```
+
+The **centreon** resource is now relocated and the cluster is OK. Check this with `crm_mon -fr` on any node.
+
+```text
+Cluster Summary:
+  * Stack: corosync
+  * Current DC: @CENTRAL_NODE1_NAME@ (version 2.1.2-4.el8_6.3-ada5c3b36e2) - partition with quorum
+  * Last updated: Wed Nov  9 10:23:54 2022
+  * Last change:  Wed Nov  9 10:23:26 2022 by root via crm_attribute on @CENTRAL_NODE1_NAME@
+  * 2 nodes configured
+  * 12 resource instances configured
+Node List:
+  * Online: [ @CENTRAL_NODE1_NAME@ centreon-rhel8-sec ]
+Full List of Resources:
+  * Clone Set: ms_mysql-clone [ms_mysql] (promotable):
+    * Masters: [ @CENTRAL_NODE1_NAME@ ]
+    * Slaves: [ centreon-rhel8-sec ]
+  * Clone Set: php-clone [php]:
+    * Started: [ @CENTRAL_NODE1_NAME@ centreon-rhel8-sec ]
+  * Clone Set: cbd_rrd-clone [cbd_rrd]:
+    * Started: [ @CENTRAL_NODE1_NAME@ centreon-rhel8-sec ]
+  * Resource Group: centreon:
+    * vip       (ocf::heartbeat:IPaddr2):        Started @CENTRAL_NODE1_NAME@
+    * http      (systemd:httpd):         Started @CENTRAL_NODE1_NAME@
+    * gorgone   (systemd:gorgoned):      Started @CENTRAL_NODE1_NAME@
+    * centreon_central_sync     (systemd:centreon-central-sync):         Started @CENTRAL_NODE1_NAME@
+    * cbd_central_broker        (systemd:cbd-sql):       Started @CENTRAL_NODE1_NAME@
+    * centengine        (systemd:centengine):    Started @CENTRAL_NODE1_NAME@
+    * centreontrapd     (systemd:centreontrapd):         Started @CENTRAL_NODE1_NAME@
+    * snmptrapd (systemd:snmptrapd):     Started @CENTRAL_NODE1_NAME@
+Migration Summary:
+```
+
+## Remove an error displayed in the cluster status
+
+Once the cause of the error has been identified and fixed (see the [troubleshooting guide](troubleshooting-guide.md)), you must manually delete the error message:
+
+```bash
+pcs resource cleanup
+```
+
+Or, if you want to remove only the errors linked to one resource:
+
+```bash
+pcs resource cleanup <resource_name>
+```
```

Comment:
This should be moved earlier in the page as this is one of the basic command that people need to know before they do other things.